docs/src/ad/movement/sccm-mecm/privilege-escalation.md

```diff
@@ -353,14 +379,14 @@ ntlmrelayx.py -t "siteDatabase.domain.local" -smb2support -socks
 
 === Windows
 
-From Windows systems, [Inveigh-Relay](https://github.com/Kevin-Robertson/Inveigh) (Powershell) can be used as an alternative to [Impacket](https://github.com/fortra/impacket)'s [ntlmrelayx.py](https://github.com/fortra/impacket/blob/master/examples/ntlmrelayx.py), however it doesn't feature the same SOCKS functionality, need in the steps detailed below, meaning the exploitation from Windows system will need to be adapted.
+From Windows systems, [Inveigh-Relay](https://github.com/Kevin-Robertson/Inveigh) (Powershell) can be used as an alternative to [Impacket](https://github.com/fortra/impacket)'s [ntlmrelayx.py](https://github.com/fortra/impacket/blob/master/examples/ntlmrelayx.py), however it does not feature the same SOCKS functionality, needed in the steps detailed below, meaning the exploitation from Windows system will need to be adapted.
 
 :::
 
 
 Fore more insight on NTLM relay attacks and tools options, see the corresponding page on The Hacker Recipes: [NTLM Relay](../ntlm/relay).
 
-3. Authentication coercion
+##### Step 3: coerce authentication
```

Comment:
The Step 4 is still marked `4. Obtain an SQL console` bellow. Can you change it to `##### Step 4: Obtain an SQL console` ?